visualstimulation/analysis.py

```diff
@@ -62,29 +63,34 @@ def compute_osi(rates, orients):
         array of firing rates
     orients : quantity array
         array of orientations
+    normalise : bool
+        Feature scaling; true or false
     Returns
     -------
     out : float
         orientation selectivity index
     """
-    from visualstimulation.helper import wrap_angle, find_nearest
+    from .helper import wrap_angle, find_nearest
+
+    if normalise is True:
+        n_rates = ( (rates - rates.min()) / (rates.max() - rates.min()) ) * pq.Hz
```

Comment:
`n_rates` is not used.

Reply:
I see in this commit e80540fbd92940f5ef32c15bcf99aeb38f257d07  that the normalization part is used several more places. Therefore this part should be moved to a separate function (e.g. in utils) that you reuse other places.

Reply:
I suggest we switch to this [function ](https://stackoverflow.com/questions/52574040/min-max-scaling-along-rows-in-numpy-array) instead